### DISCUSSION TITLE
vitepress bug？Feedback questions, please reply to me

### DISCUSSION THREAD

**Discussion by zeMinng** [vuejs/vitepress#3447] [Q&A]
My project did not load the requested vpXX file when it first started building, and now I start loading them.
 Process: I delete the theme/index.ts file, and then restore the theme/index.ts

Now crazy request vpXX, cause my project to first render too slowly.

help

or bug？

![image](https://github.com/vuejs/vitepress/assets/75962652/5acc3a1c-7736-4021-aa0b-ab4fe3cdef50)

**Comment by zeMinng** [comment]
The previous request was very clean, see Fig

![image](https://github.com/vuejs/vitepress/assets/75962652/21d370c3-4456-4fe2-9b51-1e46108a6df3)


now
![image](https://github.com/vuejs/vitepress/assets/75962652/89b6d5c9-c27a-4e51-bd49-c6f88db5f76c)

**Comment by brc-dd** [comment]
That's just how vite works in dev. It's a dev-only behavior and shouldn't affect the performance in production builds. Also, I don't think it should be slow unless some browser extension of yours is interfering with it. Can you include the time of each request in your screenshot as well? (it should be there alongside size; just shrink status, type, etc.)

**Reply by zeMinng** [reply]
As shown in the picture, after the formal packaging, it will also make crazy requests, resulting in a too long white screen loaded for the first time
![image](https://github.com/vuejs/vitepress/assets/75962652/d28f1a5d-562c-476e-93d1-4dd8786090fe)

**Reply by brc-dd** [reply]
Can you provide a reproducible example in form or some repository? Something seems wrong with your build + deploy setup though. In build, you should have just 2-3 requests.

**Reply by zeMinng** [reply]
You can pull my project, you look at my configuration。
   https://github.com/2439340964/vlog
Not xhr it is a request vpxxxx. I don t want to request vpxx these too influential

**Reply by brc-dd** [reply]
It seems to be working fine for me in `vitepress build` and `vitepress preview`:

<img width="1472" alt="image" src="https://github.com/vuejs/vitepress/assets/40380293/7e6e7338-b1a0-4df5-9a55-5ca43a668cb5">

You are probably running just `vitepress dev`.  But even on that, my requests resolve within few ms:

<img width="1462" alt="image" src="https://github.com/vuejs/vitepress/assets/40380293/841d7ea2-8472-4530-8519-37f0b9f5e601">

Try disabling some extensions or using some other browser.

**Reply by zeMinng** [reply]
Yes, yes, I mean, the second picture you sent out. There are a lot of VPxxx, which can be solved by custom theme / index, but there will be a lot of VPxx acquisitions

**Reply by zeMinng** [reply]
My appeal is: to avoid a lot of VPxxx acquisition requests

**Reply by brc-dd** [reply]
But that's only on your local computer, and all that for me resolves in <100ms.

**Reply by zeMinng** [reply]
My server response is not very timely and you can refer to this url which is the same problem as me with lots of VPxxx requests causing slow loading

http://www.lll.plus/

**Reply by brc-dd** [reply]
> http://www.lll.plus/

How have you deployed it? To me it looks like you've just run the dev and exposed the port 👀

**Reply by zeMinng** [reply]
oh，How do I send a private message，It seems like you don't know what I mean.

**Reply by brc-dd** [reply]
You can ping me on the discord - https://chat.vuejs.org/ in #vitepress channel

I understand your problem, but there is something wrong the way you've deployed your site. Otherwise there won't be that much requests.

**Reply by zeMinng** [reply]
help me😭

or。Can I modify the default no-import
![image](https://github.com/vuejs/vitepress/assets/75962652/75700d6a-78fc-4942-ac99-dcbd21e72d66)
